node-setup/080_register_stakepool.md

```diff
@@ -30,25 +30,62 @@ At this moment we have:
 
 Registering your stake pool requires:
 
+* Create JSON file with your metadata and store it in the node and in a url you maintain.
```

Comment:
"and store it in the node" ? What does this refer to?

Yes at a URL you control.